alert-common/src/main/java/com/synopsys/integration/alert/common/event/EventManager.java

```diff
@@ -23,28 +23,22 @@
 package com.synopsys.integration.alert.common.event;
 
 import java.util.List;
-import java.util.Map;
-import java.util.UUID;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.Lazy;
 import org.springframework.jms.core.JmsTemplate;
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
 import com.synopsys.integration.alert.common.ContentConverter;
-import com.synopsys.integration.alert.common.persistence.accessor.AuditUtility;
 
 @Component
 public class EventManager {
     private final JmsTemplate jmsTemplate;
-    private final AuditUtility auditUtility;
     private final ContentConverter contentConverter;
 
     @Autowired
-    public EventManager(final ContentConverter contentConverter, @Lazy final AuditUtility auditUtility, final JmsTemplate jmsTemplate) {
+    public EventManager(final ContentConverter contentConverter, final JmsTemplate jmsTemplate) {
```

Comment:
Where are we using this vanilla version of the class? Can it be abstract? Does it need to be a component still?

Reply:
This is current used for NotificationEvent's but it could be used in the future for all non-channel events